### ISSUE TITLE
Make ballot manifest info on ballot verif page visually match start-of-round page

### ISSUE THREAD

**Issue by sfsinger19103** [FreeAndFair/ColoradoRLA#571]
![image](https://user-images.githubusercontent.com/14353822/29897979-5e11b1ac-8d98-11e7-9eff-28d36a1361d3.png)
![image 1](https://user-images.githubusercontent.com/14353822/29897982-61b5da36-8d98-11e7-8d66-fc83fdc94cfd.png)

**Comment by sfsinger19103** [comment]
On ballot verification page, instead of enumerated list 

- Ballot Type: 9
- Scanner ID: 10
- Batch ID: 2
- Record ID: 7

use the same format as on the Ballots to Audit page, a table with one header row and just one entry.

![clipped](https://user-images.githubusercontent.com/14353822/29898087-cd5cfe36-8d98-11e7-859a-d24bc4b9953e.png)

**Comment by sfsinger19103** [comment]
Oops, blocked on my understanding of exactly what info is where.....

**Comment by ranweiler** [comment]
@sfsinger19103, is it correct that the ballot type *should* appear in the text, but should *not* appear in the one-row table? Or should it appear in both?

**Comment by sfsinger19103** [comment]
Correct:  the ballot type should appear in the text, but should not appear in the one-row table.

- Is "batch ID" the same as "Storage Bin #"? We should use the same name for both. 
- In the table, if the "Ballot Position" is the position within the bin, then doesn't it make more sense to reorder: Storage Bin #, then Ballot Position?
- Omit all "#" characters? I think they're obvious from context. 

In summary, table headings should be:

Scanner ID, Batch ID, Bin ID, Position in Bin

and all four columns should be used both for the full list of ballots and for the "list" of one single ballot .

**Comment by ranweiler** [comment]
> Is "batch ID" the same as "Storage Bin #"? We should use the same name for both.

Nope, these are distinct concepts. We sometimes lack storage bin #, FYI. If I use the Arapahoe test data, where the manifest matches the CVR export, we have it. With other test data, we don't, but maybe that is only because we aren't using the correct manifests. I'm not sure what we can expect here. For now, I'll render a Storage Bin if we have it, and an em dash if we don't. @nealmcb, any insight into appropriate expectations, here?

> In the table, if the "Ballot Position" is the position within the bin, then doesn't it make more sense to reorder: Storage Bin #, then Ballot Position?

I don't know if that's what "ballot position" means. @nealmcb, any insight? The value we actually have (and are rendering) is a field called `record_id` (this, along with the `scanner_id` and `batch_id` form the `imprinted_id`). I don't know if that is the same thing as "position in the bin". In terms of column order, I am following the CDOS spec as given by the mockup (e.g. described in #540). 

> Omit all "#" characters? I think they're obvious from context.

In the CDOS mockup in #540, they actually specify e.g. "Batch no.". I perhaps should've used that, but in the moment, used a `#` instead of `no.` because it looked less repetitious, while still staying close to the spec. I think it is an okay compromise, and would suggest leaving it as-is unless CDOS asks for something else. If we're going to do anything, I think changing it to "ID" is most appropriate, since that emphasizes that the value is an ID number, not a value that indicates the _count_ of something.

In summary, I'd like to either (1) leave the headers as they are, or (2) change the `#` to `ID`, not changing anything else. Either way, of course, we'll use the one-row table format in the ballot verification screen, and make it match the ballot list, however it ends up.

**Comment by dmzimmerman** [comment]
There is no such thing as "storage bin #" except when it's listed in a ballot manifest as a storage location.

**Comment by sfsinger19103** [comment]
Use your judgment. As you say, if CDOS wants something different they'll let us know.

**Comment by nealmcb** [comment]
The performance test cvr data doesn't have matching manifests, but they are easy to make - one line apiece.  In CDOS testing they should have matching manifests and thus have storage locations, except for phantom ballots and batches which in fact don't have a storage location, so marking those as "NOT_IN_MANIFEST" or something would be fine.

**Comment by nealmcb** [comment]
record_id is the sequential position in the batch. Multiple batches can be in a single bin/box/storage location.
Using "ID" instead of "#" for scanner and batch matches the spec and is fine, or staying with "#", as you like. "#" makes a bit more sense for the record_id which is in fact a sequential count of ballot cards in a batch.

**Comment by sfsinger19103** [comment]
If we can manage to have the header row exactly the same (no column omitted) between the two pages, that's a good thing. If there's a good reason not to do that, fine.

**Comment by sfsinger19103** [comment]
"That's how it is in the CDOS mockup" is a good reason, of course.

**Comment by sfsinger19103** [comment]
It seems “Storage bin” just extra information from the manifest to help in finding the ballot. If it's easy to separate it visually (double line before the column, lighter text, or something), please do.

**Comment by ranweiler** [comment]
Closed in #588. We can add the storage bin after #587 lands.
